contrib/rocketmq_proxy/filters/network/source/router/route_matcher.h

```diff
@@ -44,7 +44,7 @@ class RouteEntryImpl : public RouteEntry,
 private:
   bool headersMatch(const Http::HeaderMap& headers) const;
 
-  const Matchers::StringMatcherImpl<envoy::type::matcher::v3::StringMatcher> topic_name_;
```

Comment:
I really like that the outer class is no longer templatized on the protobuf type!